docs/examples/example.ipynb

```diff
@@ -567,14 +567,311 @@
     "    ]\n",
     "  }\n",
     "]\n",
+    "```"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create modular components\n",
+    "Make a modular resource from an existing component.\n",
+    "The steps required in creating a modular component and connecting it to a robot are\n",
+    "1. **Subclass a component and implement desired functions.** This will be your custom component.\n",
+    "    - For functions that you do not wish to implement, you must at least define by putting `pass` or `raise NotImplementedError()` in the function.\n",
+    "2. **Register a new model.** This will register the new modular resource into the `Registry`.\n",
+    "3. **Make a module executable,** This allows `viam-server` to access and execute the module.\n",
+    "4. **Configure a modular resource.** Use the new module and instantiate a new component to a robot!\n",
+    "    \n",
+    "### 1. Subclass a component and implement desired functions \n",
+    "The SDK provides a wide array of components to customize. You can browse through the API Reference to see all of them, but for now we’ll use an `Arm` as an example. Our custom Arm will be extremely simple – it will only save and return the positions provided to it.\n",
+    "\n",
+    "Let’s start by creating a directory called `my-python-robot`. Inside of that directory, create a file called `my_cool_arm.py`.\n",
```

Comment:
I would rename this to be `my-python-module` or something else to differentiate it from the custom components section. 

Similarly, change the arm name to `modular_arm` or something